planetarium/NineChronicles
# Show combination loading screen on grinding.

### Description

1. SSIA

### How to test

1. Check if combination loading screen is displaying properly on grinding.
2. Check if result currency is properly displaying in speech bubble.

### Required Reviewers
@sonohoshi 

### Screenshots
![image](https://user-images.githubusercontent.com/26648527/165522621-d252f746-b672-4515-839d-ed664efc3daa.png)
